auth/src/commonMain/kotlin/com/liftric/auth/jwt/CognitoAccessToken.kt

```diff
@@ -35,7 +37,7 @@ class CognitoAccessToken(accessTokenString: String): JWT<CognitoAccessTokenClaim
     override val claims: CognitoAccessTokenClaims
         get() {
             try {
-                return Json.decodeFromString(CognitoAccessTokenClaims.serializer(), getPayload())
+                return Json { ignoreUnknownKeys = true }.decodeFromString(CognitoAccessTokenClaims.serializer(), getPayload())
```

Comment:
This creates a new `Json` instance on each `get()` call. Better to cache it statically like this:
```
class CognitoAccessToken(accessTokenString: String): JWT<CognitoAccessTokenClaims>(accessTokenString) {
    override val claims: CognitoAccessTokenClaims
        get() {
            try {
                return json.decodeFromString(CognitoAccessTokenClaims.serializer(), getPayload())
            } catch (e: Exception) {
                throw InvalidCognitoAccessTokenException("This is not a valid access token")
            }
        }
    companion object {
        private val json by lazy {
            Json { ignoreUnknownKeys = true }
        }
    }
}
```

Reply:
That's a good catch @Ingwersaft!